apps/examples/kernel_sample/workqueue.c

```diff
@@ -40,47 +45,52 @@ static void wq_test1(FAR void *arg)
 	systime_t cur_time = 0;
 	cur_time = clock_systimer();
 
-	printf("test1 is excuted at (%d) ticks\n", cur_time);
+	wq_order[1] = (uint64_t)cur_time;
```

Comment:
We need to discuss this method. This is not a wide usage. If user modifies time value, this is definitely wrong.